### PR TITLE
Rework to use OctoKit in order to use paginate method

### PR DESCRIPTION
**Pull request by vbouchet31**
Remove the dependency to GitHub library which has been replaced by OctoKit and use paginate method in order to get all the comments instead of the first X ones only.

### CODE REVIEW

File: src/Octokit.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// MIT © 2021 vbouchet31
+"use strict";
+const { Octokit } = require("@octokit/rest");
+
+export default class OctokitClient {
+    constructor(token) {
+        this.octokit = new Octokit({
+            auth: token,
+            timeZone: "Europe/Amsterdam",
```
Comment (azu) [inline src/Octokit.js]: Probably,  it is good that `timeZone` is the default(no specify)
https://docs.github.com/en/rest/overview/resources-in-the-rest-api#timezones

**Review by azu — COMMENTED** [review]
Thanks!

LGTM without `timeZone`

**Comment by azu** [comment]
@vbouchet31 Thanks for your work!

However, I no longer use this tool.
If you are ok, I want to transfer this repository.
Also, Do you have [npm](https://www.npmjs.com/) account?

**Comment by vbouchet31** [comment]
Hi @azu . I don't have a npm account. It is my first contribution to a node project. However, given it is not a too visible and too sensitive tool, I am fine taking the risk of maintaining it if you wish. (I may regret this words later ;-) ).
I will create an account and let you know.

**Comment by vbouchet31** [comment]
https://www.npmjs.com/~vbouchet31

**Comment by azu** [comment]
Nice! I've invited you to npm and this repo.

Now, you can do `npm publish`.

**Comment by azu** [comment]
After merging this PR, you can publish a new version to npm.

## Updating Package

Almost Node.js project use [semantic versioning](https://docs.npmjs.com/about-semantic-versioning).
This PR's change is "fixes".
So, you can use `npm version patch` command to update [package's version](https://github.com/azu/get-github-pr-review-comments/blob/938764568daa1447d08148d0131e7980a3be3efd/package.json#L13).

After update the package's version, you can publish the new package to npm via `npm publish` command.

- [npm-version | npm Docs](https://docs.npmjs.com/cli/v6/commands/npm-version)
  - Fix: `npm version patch`
  - Feature: `npm version minor`
  - Breaking Change: `npm version major`
- [Updating your published package version number | npm Docs](https://docs.npmjs.com/updating-your-published-package-version-number)

Commands:

```
# you need to login to npm first time
$ npm login
# update package.json
# Following command will do 1.2.2 → 1.2.3 and git tag
$ npm version patch
# publish to npm
$ npm publish
```

**Pull request merged**